german/ruderboot.md

```diff
@@ -1,8 +1,8 @@
-Sie wachen von der Sonne geblendet auf und stellen ziemlich benommen fest, dass Sie 
+Sie wachen von der Sonne geblendet auf und stellen ziemlich benommen fest, dass Sie
 
 in einem Ruderboot auf einem grossen See treiben. Ihre Hose ist vollkommen durchn�sst, und
 
-Sie bemerken, dass durch ein Loch in der Bootswand Wasser ins Boot dringt.
+Sie bemerken, dass durch ein Loch in der Bootswand Wasser ins Boot dringt. Mehrere grosse Haiflossen schwimmen um das Boot herum und dass Wasser brodelt verd�chtig.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.